_posts/2018-01-19-event-lineup.md

```diff
@@ -0,0 +1,76 @@
+
+Several workshops are already scheduled for the beginning of February, with hopefully more to come.
+Find all information on [the Roadshow website][roadshow].
+
+[roadshow]: https://rust-br.github.io/2018-roadshow/
+
+### February 3-4: FOSDEM - Rust Dev Room
+
+After a very successful Bird of Feathers session for Rust last year at FOSDEM,
+this year will have the first edition of the [Rust Dev Room][fosdem-rust].
+Co-located with [FOSDEM 2018][fosdem], the annual conference about free and open source software,
+it will bring together the Rust community in **Brussel, Belgium**.
+
+A fully packed Sunday brings you 16 talks covering topics ranging from GUI programming, robotics and multimedia to idiomatic Rust, testing and memory management.
+Check out [the full devromm schedule][devroom-schedule].
+
+[fosdem]: https://fosdem.org/2018/
+[fosdem-rust]: https://rust-fosdem.github.io/
+[devroom-schedule]: https://fosdem.org/2018/schedule/track/rust/
+
+### May XX-YY: RustFest Paris
+
+The fourth iteration of RustFest will happen on May XXth and YYth in **Paris, France**.
+Once again there will be a full day of talks and a second day with multiple workshops, allowing you to learn, connect and code together with other Rustaceans.
+
+The CfP and ticket sales will open soon.
+Keep an eye on [rustfest.eu], get all updates on the [blog][rustfest-blog] and don’t forget to follow [@rustfest] on Twitter.
+Want to get a glimpse into what it's like? Check out last year's videos from [Kyiv] or [Zurich]!
+
+[rustfest.eu]: http://rustfest.eu
+[rustfest-blog]: http://blog.rustfest.eu/
+[@rustfest]: https://twitter.com/rustfest
+[kyiv]: https://www.youtube.com/watch?v=AHprJNUCgQ0&list=PL85XCvVPmGQhvs1Rnet_24B-AI3YSM2YG
+[zurich]: https://www.youtube.com/watch?v=jywiVWKm1TI&list=PL85XCvVPmGQj9mqbJizw-zi-EhcpS5jTP
+
+### August 16-17: RustConf
+
+[RustConf] is a two-day event held in **Portland, OR, USA** on August 16-17.
+
+The first day will again offer tutorials on Rust given directly by members of the Rust core team, ranging from absolute basics to advanced ownership techniques.
+The second day is the main event, with talks at every level of expertise, so don't miss the CfP to submit your talk soon!
+
+More details will be available on [rustconf.com][rustconf]. You can also follow [@rustconf] on Twitter for all announcements.
+
+[rustconf]: http://rustconf.com/
+[@rustconf]: https:/twitter.com/rustconf
+
+### October: Rust Belt Rust
+
+[Rust Belt Rust](https://www.rust-belt-rust.com/) will have its third iteration this year in October.
+You can once again expect two days of talks and workshops.
+
+More details will come soon.
+Follow [@rustbeltrust] on Twitter for the latest news.
+
+[@rustbeltrust]: https://twitter.com/rustbeltrust
+
+---
+
+As always, there are nearly a hundred [Rust User Groups][usergroups] worldwide.
+Take a look if there's one near you or get people together and start a new one!
+
```

Comment:
might be neat to add some instructions here for how to get your event listed on our calendar or spotlighted in posts like this 

cc @Manishearth if you wanted to start a Switchboard team repo and have instructions there, that might be nice!

Reply:
Good idea, I'll try to do this this week